xarray/backends/netCDF4_.py

```diff
@@ -194,6 +194,13 @@ def _nc4_require_group(ds, group, mode, create_group=_netcdf4_create_group):
         return ds
 
 
+def _ensure_no_forward_slash_in_name(name):
+    if "/" in name:
```

Comment:
I'm not sure what the type of name is at this stage. Could it be `None` or non-str `Hashable`?

Reply:
Ah, good catch. If that's the case, we'll have to adapt that check. I'll try to get behind it.

Reply:
@headtr1ck From what I can tell, the names should already be strings at that point of the backend code. So this should be safe.